AndroidAnnotations/androidannotations-core/androidannotations-test/src/test/java/org/androidannotations/test/efragment/MyListFragmentTest.java

```diff
@@ -123,6 +123,28 @@ public void ignoredWhenDetachedWorksForIgnoredMethod() {
 	}
 
 	@Test
+	public void ignoredWhenViewDestroyedForIgnoredMethod() {
+		assertFalse(myListFragment.didExecute);
+		myListFragment.onDestroyView();
+		myListFragment.ignoreWhenViewDestroyed();
+		assertFalse(myListFragment.didExecute);
+	}
+
+	@Test
+	public void notIgnoredBeforeDetached() {
+		assertFalse(myListFragment.didExecute);
+		myListFragment.ignored();
+		assertTrue(myListFragment.didExecute);
+	}
+
+	@Test
+	public void notIgnoredBeforeViewDestroyed() {
+		assertFalse(myListFragment.didExecute);
+		myListFragment.ignoreWhenViewDestroyed();
+		assertTrue(myListFragment.didExecute);
+	}
+
```

Comment:
We need another test case here: call `onDestroyView()`, then `onCreateView()`. In that case the annotated method should not be ignored.